e2e/test/scenarios/visualizations-charts/line-bar-tooltips.cy.spec.js

```diff
@@ -517,6 +517,31 @@ describe("scenarios > visualizations > line/bar chart > tooltips", () => {
   });
 
   describe("> single series question grouped by month on dashboard", () => {
+    it("it should not apply formatting to the percent change", () => {
```

Comment:
Is it worth copying this test over to master as well, if it's not too much effort? I know we didn't see this regression on master, but it can't hurt to have extra coverage to prevent it from accidentally being introduced in the future.

Reply:
Good point, I will add it to master as well